frontend/src/components/Pages/MissionPage/VideoStream/FullScreenVideo.tsx

```diff
@@ -64,7 +54,7 @@ export const FullScreenVideoStreamCard = ({
 
     // Rotated stream is not supported for simpleplayer
     return (
-        <FullScreenCard style={{ boxShadow: tokens.elevation.raised, width: cardWidth() }}>
+        <FullScreenCard style={{ boxShadow: tokens.elevation.raised, width: '100%', backgroundColor: 'white' }}>
```

Comment:
I would move this up to the card - line 25 

const FullScreenCard = styled.div`
    box-shadow: ${tokens.elevation.raised};
    width: 100%;
    background-color: white;
    padding: 1rem;
`